internal/state/config/options.go

```diff
@@ -0,0 +1,278 @@
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"github.com/hetznercloud/cli/internal/cmd/util"
+	"github.com/hetznercloud/hcloud-go/v2/hcloud"
+)
+
+type OptionFlag int
```

Comment:
Should we rename this to `OptionFeature`? I have and will be confused by this naming more than once.

e.g. `!o.HasFlag(OptionFlagEnv)` => `!o.HasFeature(OptionFeatureEnv)`

Reply:
Alright, maybe with some getting used to I'll get the hand of it.